### ISSUE TITLE
should add a purge_queue function

### ISSUE THREAD

**Issue by einhverfr** [rpdelaney/pg-message-queue#2]
Request for discussion first.  Thinking is that the function should purge messages which a) have been delivered and b) are older than a supplied value.

**Comment by rpdelaney** [comment]
Yeah, don't see why not. We have a function to drop the whole queue, so why not a function to truncate it?
